radix-engine/src/updates/babylon.rs

```diff
@@ -167,7 +181,7 @@ impl BabylonSettings {
         Self {
             genesis_data_chunks,
             genesis_epoch: Epoch::of(1),
-            consensus_manager_config: ConsensusManagerConfig::test_default(),
+            consensus_manager_config: ConsensusManagerConfig::mainnet_genesis(),
```

Comment:
Why do we need to change this? This seems to break all scenario assertions.

Reply:
Was a suggestion from me here: https://github.com/radixdlt/radixdlt-scrypto/pull/1928#discussion_r1777619337

But I actually meant change in `dumper.rs` > `run_all_protocol_updates`, change:
```
        let babylon_settings = BabylonSettings::test_complex()
.with_consensus_manager_config(ConsensusManagerConfig::mainnet_genesis());
        let protocol_executor = ProtocolBuilder::for_network(&network_definition)
            .configure_babylon(|_| babylon_settings)
            .from_bootstrap_to_latest();
```

This will then only change it for the protocol update tests, so we can get more realistic protocol change receipts.